internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -70,6 +72,15 @@ func NewEnvStackConfig(input *deploy.CreateEnvironmentInput) *EnvStackConfig {
 	}
 }
 
+// NewEnvConfigFromExistingStack returns a CloudFormation stack configuration for updating an environment.
+func NewEnvConfigFromExistingStack(in *deploy.CreateEnvironmentInput, prevForceUpdateID string) *EnvStackConfig {
```

Comment:
i feel like calling it `prevForceUpdateID` here (and in the `EnvStackConfig` struct) is a little confusing, bc it is the current forceUpdateID that the env stack is currently on. i see what you mean though😊 

maybe just change it to just `ForceUpdateID`?

Reply:
what do ya think of `lastForceUpdateID`? I get that `prev` sounds like it's not going to be the same value anymore though in reality it depends on `in.ForceUpdate`. But maybe `latest` sounds better?

Reply:
i think `last` is perf 👌